easybuild/easyconfigs/h/html5lib/html5lib-1.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonBundle'
+
+name = 'html5lib'
```

Comment:
Note: this easyconfig was removed in #18698 because `html5lib` is already included in `Python-bundle-PyPI/2023.06-GCCcore-12.3.0`, cfr. #18208